.travis.yml

```diff
@@ -9,18 +9,13 @@ cache:
     - "$HOME/.sonar/cache"
     - "$HOME/.sonar/native-sonar-scanner"
     - "$HOME/optaweb-vehicle-routing/optaweb-vehicle-routing-frontend/node"
-# Change Travis install phase to only resolve dependencies needed to run the build.
-# Otherwise it would run `mvn install`, which results in doing some Maven phases twice (validate, compile, jar, ...).
-# https://docs.travis-ci.com/user/languages/java/#projects-using-maven
-#
-# We also want to prevent Maven's install phase from writing to ~/.m2/repository, which is cached.
-install: mvn dependency:go-offline -Pcode-coverage,sonarcloud-analysis --show-version
+install: mvn clean install -DskipTests=true -DskipITs -Dmaven.javadoc.skip=true -B -V
```

Comment:
Let's not build twice (even if tests are skipped in this rounds there would be a significant overlap between the two build). Either use Christopher's solution https://github.com/kiegroup/optaweb-employee-rostering/pull/294/commits/390ff69aade072c9ff68bebcc35d6331f6c35b80 or skip install phase completely by changing to something like [install: true](https://stackoverflow.com/a/33527978/1691152).